### DISCUSSION TITLE
Presence based on image_processing entities

### DISCUSSION THREAD

**Discussion by grinco** [jseidl/magic-areas#184] [Ideas / Feature Requests]
You should include image_processing entities to calculate presence. I'm using it to detect people in the camera streams. Is useful when watching tv for example, or reading, when there is no or limited motion for the motions sensors to pick it up.

**Comment by jseidl** [comment]
Wasn't aware of this integration, nice! Although it seems to only fire events, not having an entity for us to monitor. Probably this would best be paired with the new trigger based binary_sensors https://www.home-assistant.io/integrations/template/#configuration-for-trigger-based-template-sensors

**Comment by grinco** [comment]
That's how I use it, yes. Have binary sensors which are triggered when
number of people in the image is > 0, then binary sensors are assigned to
the magic areas.
